src/main/java/com/example/HelloWorldApplication/FormController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.HelloWorldApplication;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Controller
+public class FormController {
+
+    @GetMapping("/form")
+    public String init(Model model) {
+        model.addAttribute("moji", "Hello World!");
+        //指定された名前に指定された属性を追加する。
+
+        String now = LocalDateTime.now().format( DateTimeFormatter.ofPattern("yyyy年MM月dd日 HH時mm分ss秒 E曜日")); // ②
+        model.addAttribute( "datetime", now );
```

Comment:
フォーマット設定がOFFになっていると思います。
※プロジェクトごとに設定しないといけないんですよね・・・。
```suggestion
        model.addAttribute("datetime", now );
```